docs/submit-transactions.md

```diff
@@ -0,0 +1,149 @@
+        radicle-registry-cli org unregister --help
+        ```
+
+### Examples
+
+1. Register a user
+
+    This example registers a user named "neo" in the registry using a
+    locally generated account (that happens to also be) named "neo"
+    to sign the transactions, specifying `100` as the fee.
+
+    ``` bash
+    radicle-registry-cli user register neo --author neo --fee 100
+    ```
+
+2. Transfer funds from an account to another
+
+    This example transfers `500` tokens to the account with the address
+    `5HWP48i9TuP2VrRZrNeb6QzYpdUSE9BvyaptEqqHrLfH8ZPd`, specifying `20` as fee. A total of
+    `520` will be withdrawn from the author's account, neo.
+
+    ``` bash
+    radicle-registry-cli account transfer 500 5HWP48i9TuP2VrRZrNeb6QzYpdUSE9BvyaptEqqHrLfH8ZPd --author neo --fee 20
+    ```
+
+3. Unregister an org
+
+    This example unregisters an org named "Hooli". For it to succeed, such an org must be registered in
+the Radicle Registry and it must have been registered by the author. The author will be charged
+    `10` tokens in any case.
+
+    ``` bash
+    radicle-registry-cli org unregister Hooli --author neo --fee 10
+    ```
+
+## Command-Line Options
+
+All the CLI commands that submit transactions provide the following options that
+are to be specified at the end of your command.
+
+- `Author`
+
+    ⚠ Requires having [getting started submitting a transaction](getting-started#submitting-a-transaction).
+
+    Mandatory for all transactions, the author option lets you specify the local account
+    that you want to use to sign a transaction.
+
+    ```
+    --author <account_name>
+        The name of the local account to be used to sign transactions
+        [env: RAD_AUTHOR=]
+    ```
+
+- `Fee`
+
+    ⓘ See [Transaction fees](#transaction-fees) to learn more about how fees work in the Radicle Registry.
+
+    Optional, default to the minimum acceptable fee amount.
+
+    ```
+    --fee <fee>
+        Fee that will be charged to submit transactions. The higher the fee,
+        the higher the priority of a transaction
+        [env: RAD_FEE=]
+        [default: 1]
+    ```
+
+- `Node Host`
+
+    ⓘ See [Running a node](getting-started#running-a-node) to be able to connect directly to
+    the node you eventually have running locally.
+
+    Optional, default to the ffnet network. TODO(nuno) phrase this better.
+
+    ```
+    --node-host <node-host>
+        IP address or domain name that hosts the RPC API
+        [env: RAD_NODE_HOST=]
+        [default: 127.0.0.1]
+    ```
+
+## Transaction fees
+
+Transactions fees in the Radicle Registry let you specify the amount that you want to
+pay to have your transactions included and prioritized. We only set a minimum amount,
+you define how much we should charge you. It's surprise-free.
+
+Also note that transactions that involve an Org will be charged from the org's account,
+unless if the author is not a member of that org. In that case, the author is charged.
```

Comment:
```suggestion
Also note that transactions that involve an organization will be charged from the org's account
if the author is a member of the organization. Otherwise the transaction will fail and the author
is charged the fee.
```

We should also point out that the author still is charged for registering an org.